cmd/main.go

```diff
@@ -30,26 +30,25 @@ func main() {
 	result, err := apinighthawk.NighthawkRun(testConfig)
 
 	if err != nil {
-		msg := "Failed to run load-test"
-		err = errors.Wrapf(err, msg)
-		log.Fatal(err)
+		log.Error("Failed to run load test: ", err)
+		return
```

Comment:
What will happen after `return`? Will it also terminate the program like `log.Fatal`?

Reply:
Nope unlike `log.Fatal()`, `log.Error()` will log the error message but allows the program to continue execution. This way, even if one error occurs, the program can continue to process the rest of the results, and the error messages will help in troubleshooting.

Reply:
Sorry for the late response, understand that log.Error() will log the error message but allows the program to continue execution. However, if you add `return` after log.Error(), it will still exit the program, right? Please correct me if I'm wrong.

Reply:
Yes, you are right. Adding a `return` statement to the function will still exit the program. However, using `log.Error()` instead of `log.Fatal()` allows the program to continue executing if there is an error. 

When we call `log.Error()`, the function only prints the error message to the standard error stream. The program can continue executing after this error, but you'll have to handle the error and decide what to do next.

But in `log.fatal()`, the function prints the error message to the standard error stream and then calls os.Exit(1), which terminates the program immediately.

Reply:
But I'll say this `log.fatal()` is useful when the error is severe enough that the program can't continue executing.

Reply:
@ShivangShandilya Thanks, agree with you. LGTM.

Reply:
Thanks you gents.